git-storage/t/src/properties/odb.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2022 The Radicle Link Contributors
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+use git_storage::{
+    odb::{Read, Write},
```

Comment:
You can import like this to avoid the name clashes :)
```suggestion
    odb::{Read as _, Write as _},
```